client/disputes/details/actions.tsx

```diff
@@ -5,14 +5,25 @@
  */
 import { __ } from '@wordpress/i18n';
 import { Button } from '@wordpress/components';
+import React from 'react';
 
 /**
  * Internal dependencies
  */
 import wcpayTracks from 'tracks';
 import { getAdminUrl } from 'wcpay/utils';
 
-const Actions = ( { id, needsResponse, isSubmitted, onAccept } ) => {
+const Actions = ( {
+	id,
+	needsResponse,
+	isSubmitted,
+	onAccept,
+}: {
+	id: string;
+	needsResponse?: boolean;
+	isSubmitted?: boolean;
+	onAccept: () => void;
+} ): JSX.Element | null => {
 	if ( ! needsResponse && ! isSubmitted ) {
 		return null;
```

Comment:
Instead of null it is possible to return an empty `Fragment` i.e:
```suggestion
		return <></>;
```
It won't add anything to the DOM and allows you to specify the return type of the function as `JSX.Element` (without null)